src/vs/workbench/services/textfile/browser/textFileService.ts

```diff
@@ -462,7 +462,9 @@ export abstract class AbstractTextFileService extends Disposable implements ITex
 	}
 
 	suggestFilename(mode: string, untitledName: string) {
-		const extension = this.modeService.getExtensions(mode)[0];
+		// {{SQL Carbon Edit}} - pass language not mode for getting the extension.
```

Comment:
could you please update the tag to `// {{SQL CARBON EDIT}}` so Find in Files will work correctly with case-sensitive matching?